cli/lib/fetch.js

```diff
@@ -9,22 +9,19 @@ const fetch = require('node-fetch');
 const https = require('https');
 
 const httpAgent = new http.Agent({
-    family: 6, hints: dns.ADDRCONFIG | dns.V4MAPPED
-})
+  family: 6,
+  hints: dns.ADDRCONFIG | dns.V4MAPPED
+});
 const httpsAgent = new https.Agent({
-    family: 6, hints: dns.ADDRCONFIG | dns.V4MAPPED
-})
+  family: 6,
+  hints: dns.ADDRCONFIG | dns.V4MAPPED
+});
 
 function fetchWithAgent(resource, init) {
```

Comment:
This line could be `function fetchWithAgent(resource, init = {}) {` rather than including the check below.